CRM/Contact/Page/DedupeFind.php

```diff
@@ -149,7 +149,7 @@ public function run() {
 
       $stats = CRM_Dedupe_Merger::getMergeStatsMsg($cacheKeyString);
       if ($stats) {
-        CRM_Core_Session::setStatus($stats);
+        CRM_Core_Session::setStatus($stats, '', 'alert', array('expires' => 0));
```

Comment:
How about giving it a title as well, like "Batch Complete". Also, is this a success message?